.Rbuildignore

```diff
@@ -54,4 +54,4 @@
 # Other files related to package development.
 
 ^LICENSE\.md$
-# Required by the GNU GPL license but not allowed on CRAN
+# Required to be in the repository but not allowed on CRAN
```

Comment:
I added a `LICENSE` template file to the skeleton which produces a similar file to the `biocro-dev` `LICENSE` file in PR #510. I'm including the separate `LICENSE.md` file with the text of the MIT license because that's how `usethis::use_mit_license()` does it.